qutebrowser/html/history.html

```diff
@@ -16,43 +16,71 @@
     white-space: nowrap;
 }
 
-.date {
-    color: #888;
-    font-size: 14pt;
-    padding-left: 25px;
+table {
+    margin-bottom: 30px;
 }
 
-.pagination-link {
-    display: inline-block;
-    margin-bottom: 10px;
-    margin-top: 10px;
-    padding-right: 10px;
+.date {
+    color: #555;
+    font-size: 12pt;
+    padding-bottom: 15px;
+    font-weight: bold;
+    text-align: left;
 }
 
-.pagination-link > a {
-    color: #333;
+#load {
+    color: #555;
     font-weight: bold;
+    text-decoration: none;
 }
-{% endblock %}
 
-{% block content %}
+#eof {
+    color: #aaa;
+    margin-bottom: 30px;
+    text-align: center;
+    width: 100%;
+}
 
-<h1>Browsing history <span class="date">{{curr_date.strftime("%a, %d %B %Y")}}</span></h1>
+.session-separator {
+    color: #aaa;
+    height: 40px;
+    text-align: center;
+}
 
-<table>
-    <tbody>
-    {% for url, title, time in history %}
-        <tr>
-            <td class="title"><a href="{{url}}">{{title}}</a></td>
-            <td class="time">{{time}}</td>
-        </tr>
-    {% endfor %}
-    </tbody>
-</table>
+.error {
+    background-color: #ffbbbb;
+    border-radius: 5px;
+    font-weight: bold;
+    padding: 10px;
+    text-align: center;
+    width: 100%;
+    border: 1px solid #ff7777;
+}
+
+{% endblock %}
+{% block content %}
+<h1>Browsing history</h1>
+<noscript>
+    <div class="error">Javascript is required to view this page.</div>
+</noscript>
+<div id="hist-container"></div>
+<span id="eof" style="display: none">end</span>
+<a href="#" id="load">Show more</a>
```

Comment:
This also shows up when showing the `<noscript>` error, which is a bit confusing.